### PR TITLE
first version of the db schema

### PR DESCRIPTION
**Pull request by tiikerikakku**

### CODE REVIEW

File: src/db_helper.py [hunk 1]
```diff
@@ -1,48 +1,53 @@
...
 
-  print(f"Creating table {table_name}")
+  print(f'Creating table {table_name}')
   sql = text(
-    f"CREATE TABLE {table_name} ("
-    "  id SERIAL PRIMARY KEY, "
-    "  content TEXT NOT NULL,"
-    "  done BOOLEAN DEFAULT FALSE"
-    ")"
+    f'create table {table_name} ('
```
Comment (kaltionkasvatti) [inline src/db_helper.py:36]: We could still use capitalisation in the function names, like CREATE TABLE and such,

File: src/db_helper.py [hunk 2]
```diff
@@ -1,48 +1,53 @@
...
   sql_table_existence = text(
-    "SELECT EXISTS ("
-    "  SELECT 1"
-    "  FROM information_schema.tables"
-    f" WHERE table_name = '{name}'"
-    ")"
+    'SELECT EXISTS ('
+    '  SELECT 1'
+    '  FROM information_schema.tables'
+    f' WHERE table_name = \'{name}\''
```
Comment (kaltionkasvatti) [inline src/db_helper.py:11]: No opposition to using backslash, just wondering why it is there?

**Review by kaltionkasvatti — CHANGES_REQUESTED** [review]
For integrity's sake, let's write SQL commands with appropriate capitalisation

File: src/db_helper.py (hunk 2 above)
Comment (tiikerikakku) [reply src/db_helper.py:11]: using single quotes in python, so need to escape it in the sql string

**Pull request merged**